docker/local_development/monitoring-secrets/zagg-server-vars.yml

```diff
@@ -0,0 +1,20 @@
+---
+g_default_zagg_server_config:
+  targets:
+  - name: local cluster zbx server
+    type: zabbix
+    trapper_server: oso-cent7-zabbix-server
+    trapper_port: 10051
+    api_url: http://oso-centos7-zabbix-web/zabbix/api_jsonrpc.php
+    api_user: Admin
+    api_password: 'zabbix'
+    path: /tmp/zagg/data/cluster-zbx
+  templates:
+  - Template Heartbeat
+  - Template Zagg Server
+  hostgroups:
+  - local-dev
+
+g_default_zagg_server_user: admin
+g_default_zagg_server_password: supersecret
```

Comment:
Just following how we do our secrets in production, but this could probably be done as a template (a secret template?).